src/seth/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.8.2] - 2019-07-05
+### Fixed
+ - ABI JSON blob creation would fail when function arguments required specifying
```

Comment:
This could be improved, looking for poetic help.

Reply:
How about: 

> ABI JSON blob creation used to fail when function arguments had to be specified

Reply:
The sentence wasn't finished :)

An improvement could be "when function arguments were values of reference type`

Reply:
- ABI JSON blob creation would fail for some solc reference types, e.g. `bytes memory`, since solc 0.5.0. solc was removed as a dependency and replaced with ethers.js